tests/foreman/ui/test_oscapcontent.py

```diff
@@ -0,0 +1,161 @@
+from robottelo.helpers import get_data_file
+from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context, make_oscapcontent, make_org
+from robottelo.ui.locators import common_locators
+from robottelo.ui.session import Session
+
+
+class OpenScapContent(UITestCase):
+    """Implements Oscap Content tests in UI."""
+
+    @classmethod
+    def setUpClass(cls):
+        super(OpenScapContent, cls).setUpClass()
+        path = settings.oscap.content_path
+        cls.content_path = get_data_file(path)
+
+    @tier1
+    def test_positive_create(self):
+        """@Test: Create OpenScap content.
+
+        @Feature: OpenScap - Positive Create.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+
+        @Assert: Whether creating  content for OpenScap is successful.
+        """
+        with Session(self.browser) as session:
+            for content_name in valid_data_list():
+                with self.subTest(content_name):
+                    set_context(session, org=ANY_CONTEXT['org'])
+                    session.nav.go_to_oscap_content()
+                    make_oscapcontent(
+                        session,
+                        name=content_name,
+                        content_path=self.content_path,
+                    )
+                    self.assertIsNotNone(
+                        self.oscapcontent.search(content_name))
+
+    @tier1
+    @skip_if_bug_open('bugzilla', 1289571)
+    def test_negative_create(self):
+        """@Test: Create OpenScap content with negative values
+
+        @Feature: OpenScap - Negative Create.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+
+        @Assert: Whether creating  content for OpenScap is not successful.
+        """
+        with Session(self.browser) as session:
+            for content_name in invalid_values_list(interface='ui'):
+                with self.subTest(content_name):
+                    set_context(session, org=ANY_CONTEXT['org'])
+                    session.nav.go_to_oscap_content()
+                    make_oscapcontent(
+                        session,
+                        name=content_name,
+                        content_path=self.content_path,
+                    )
+                    self.assertIsNotNone(session.nav.wait_until_element(
+                        common_locators['haserror']))
+
+    @tier1
+    def test_positive_default(self):
+        """@Test: Check whether OpenScap content exists by default.
+
+        @Feature: OpenScap - Test Default Content.
+
+        @Steps:
+
+        1. Navigate to oscap Content page.
+        2. Set Org as Any Org.
+
+        @Assert: Whether oscap content exists by default.
+        """
+        with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            session.nav.go_to_oscap_content()
```

Comment:
Search functionality usually does navigation, so that line seems as redundant one